packages/env/lib/download-sources.js

```diff
@@ -123,16 +123,11 @@ async function downloadGitSource( source, { onProgress, spinner, debug } ) {
 		await git.clone( source.url, source.clonePath, {
```

Comment:
> Perhaps restoring the lines that were removed would then make it work on both new repositories and existing repositories.

Yeah, it's worth a shot! I'd add it in the if/else statement right above here (in `isRepo`)